public/core.d.ts

```diff
@@ -6416,6 +6416,204 @@ declare namespace JMap {
+         *     params.addFeaturesToLayerSelection(4, [{
+         *       id: 58,
+         *       properties: {JMAP_FID: 58, NOM_QR: "Bois-Francs"},
+         *       type: "Feature",
+         *       geometry: {coordinates: [],type: "Polygon"}
+         *     }])
+         *     console.log("Mouseover selection after", params.selection[4])
+         *   }
+         * )
+         * ```
+         * 
+         * This listener add in every click on the map the feature, so everywhere you click
+         * on the map the mouseover will display on the map, even on an area having no feature.
+         * 
+         * Then paste this in the console to remove the previous listener:
+         * 
+         * ```ts
+         * JMap.Event.MouseOver.remove(“my-listener“)
+         * ```
+         * 
+         * You can test that now no mouseover is displayed anymore when we click on an empty area.
+         * 
+         * Finally you can test the event function removeFeaturesFromLayerSelection 
+         * by pasting the following code snippet:
+         * 
+         * ```ts
+         * JMap.Event.MouseOver.on.beforeContentProcessed(
+         *   "my-listener",
+         *   params => {
+         *     console.log("Mouseover selection before", params.selection[4])
+         *     params.removeFeaturesFromLayerSelection(4, [58])
+         *     console.log("Mouseover selection after", params.selection[4])
+         *   }
+         * )
+         * ```
+         * 
+         * Now you can click on the feature id=58, but no mouseover will display for if, as it is automatically removed by the listener.
+         * 
+         * @param listenerId Your listener id (must be unique for all mouseover events)
+         * @param fn Your listener function
+         * @example ```ts
+         * 
+         * // Each time the map is clicked and "layer" interactor is active
+         * JMap.Event.MouseOver.on.beforeContentProcessed("my-listener", params => {
+         *   // mouseover "clicked" features for layer id="3"
+         *   let features = params.getFeaturesByLayerId(3) // returns an array copy
+         *   // remove feature id=17
+         *   params.removeFeaturesByLayerId(3, [123, 432])
+         *   features = params.getFeaturesByLayerId(3)
+         *   // now features id=123 and 432 are not anymore in features array
+         *   const newFeature = { id="24553", ...etc } // a geojson feature
+         *   params.addFeatureByLayerId(3, newFeature)
+         *   features = params.getFeaturesByLayerId(3)
+         *   // features contains the new feature and will display it in the mouseover
+         * })
+         * ```
+         */
+        function beforeContentProcessed(listenerId: string, fn: (params: JMouseOverBeforeEventParams) => void): void
+        
+        /**
+         * ***JMap.Event.MouseOver.on.afterContentProcessed***
+         * 
+         * This event is triggered when the map has been clicked, before the mouseover content is calculated.
```

Comment:
This event is triggered when the map has been clicked, _**after**_ the mouseover content is calculated.

C'est pas dit dans la doc, mais j'imagine que c'est ici qu'on peux manipuler le content? Dans le cas d'un mouse over "vide" (dans une area empty de la carte), est-ce qu'on pourrait se servir de cet event pour ajouter du contenu dans le popover?